### PR TITLE
Update embedded social media, fire button & cookie popup sections

### PR DESCRIPTION
**Pull request by laghee**
Asana tasks:
- https://app.asana.com/0/1202899525821663/1205009431190155/f
- https://app.asana.com/0/1202899525821663/1204129472030626/f

Add Fire button capabilities to 3 extensions, update embedded social media protection name, clean up some missed cookie pop-up edits.

### CODE REVIEW

**Review by kzar — CHANGES_REQUESTED** [review]
GitHub doesn't allow me to leave an inline comment for an unchanged line (yay....), but line 189 has one last instance of "Embedded Social Content Tracking" @laghee

**Comment by laghee** [comment]
Thanks, @kzar -- totally missed that one 🙏

**Review by kzar — APPROVED** [review]
Click to Load changes LGTM Kate :+1: 

(I've not reviewed the other changes. Maybe wait for Konrad to review those, or if you'd like me to take a pass, could you share the document/task with those copy changes?)

File: _docs/privacy/web-tracking-protections.md [hunk 1]
```diff
@@ -292,16 +292,16 @@ We review user-reported Broken Site Reports daily, and work to address these usa
...
 | Firefox extension | Unsupported                                                                                                                                               |
-| Chrome extension  | Unsupported                                                                                                                                               |
-| Edge extension    | Unsupported                                                                                                                                               |
-| Opera extension   | Unsupported                                                                                                                                               |
+| Chrome extension  | Clears all browsing history and data. Can also clear data for a single site or for time periods. History and tab clearing can be disabled in Settings.                                                                                                                                   |
```
Comment (kdzwinel) [inline _docs/privacy/web-tracking-protections.md:302]: hm, should we also say that extensions don't support fireproofing for now? Or is it enough that we hint it out in the main description above?

**Review by kdzwinel — APPROVED** [review]
I left one comment for your consideration @laghee . Feel free to merge.

File: _docs/privacy/web-tracking-protections.md (hunk 1 above)
Comment (laghee) [reply _docs/privacy/web-tracking-protections.md:302]: We don't specify that we **do** support it in the table entries for the apps, so I think the description explanation is probably enough here. Any objections, @alohaas ?

Comment (alohaas) [reply _docs/privacy/web-tracking-protections.md:302]: Agree with @laghee. Let's merge now so we don't fall too far behind the rollout. Description is clear about support, but we can revisit additional mentions in the table and do a follow-up if needed.

**Pull request merged**